tutorials/log.md

```diff
@@ -99,6 +99,9 @@ directory specified to record:
 
 ### From plugin in SDF
 
+This feature is deprecated and will be removed in Ignition Dome.
+Use the command line argument instead.
+
```

Comment:
I think we should leave this out of the forward-port. @mabelzhang is looking into the documentation updates specific to Dome, see #293.

Reply:
I need to continue that task. I'll make a PR to Dome directly when I find out what's up with those tests.

Reply:
This relates to the comment I left in the description of this PR:

> One thing to note about this forward port is that `examples/worlds/log_playback.sdf` exists in `ign-gazebo3`, but not `ign-gazebo4`. #386 details why this file was removed. Also, is the `examples/worlds/log_playback.sdf` file in `ign-gazebo3` correct, or does that also need to be updated?

Do you guys know the answer to the question about `ign-gazebo3` in that comment?

Also, just to be clear, should I go ahead and remove the lines @chapulina mentioned for this forward port and let @mabelzhang address this when she makes a PR to Dome?

Reply:
That SDF file should exist in ign-gazebo3 (with a deprecation note), and should not exist in ign-gazebo4, in other words, leaving it in its current state in both. The lines in the tutorial can be removed. I have a different line for Dome in a branch somewhere. Thanks!

Reply:
I went ahead and made the change in https://github.com/ignitionrobotics/ign-gazebo/pull/459/commits/08f1a0f646b8b492a3a5bc1cc67136fea0b3b75f, can you guys give it one last look?